.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"google_sign_in","dependencies":[]}]}
```

Comment:
lol you should follow the instructions

Reply:
@SamChou19815 I did not change this. This is probably from when it was first generated.

Reply:
I mean it should be gitignored.